suite-common/wallet-core/src/fiat-rates/fiatRatesMiddleware.ts

```diff
@@ -1,76 +1,39 @@
+import { isAnyOf } from '@reduxjs/toolkit';
+
+import { isNative } from '@trezor/env-utils';
 import { createMiddlewareWithExtraDeps } from '@suite-common/redux-utils';
-import { BLOCKCHAIN as TREZOR_CONNECT_BLOCKCHAIN_ACTIONS } from '@trezor/connect';
-import { isTickerSupportedByBlockbook } from '@suite-common/fiat-services';
 
-import { transactionsActions } from '../transactions/transactionsActions';
-import { accountsActions } from '../accounts/accountsActions';
-import { selectAccounts } from '../accounts/accountsReducer';
 import {
-    removeFiatRatesForDisabledNetworksThunk,
-    updateCurrentFiatRatesThunk,
-    initFiatRatesThunk,
-    updateLastWeekFiatRatesThunk,
+    fetchFiatRatesThunk,
+    updateMissingTxFiatRatesThunk,
     updateTxsFiatRatesThunk,
-    onUpdateFiatRateThunk,
 } from './fiatRatesThunks';
 import { blockchainActions } from '../blockchain/blockchainActions';
-import { selectAccountTransactions } from '../transactions/transactionsReducer';
+import { accountsActions } from '../accounts/accountsActions';
+import { transactionsActions } from '../transactions/transactionsActions';
 
 export const prepareFiatRatesMiddleware = createMiddlewareWithExtraDeps(
     (action, { dispatch, extra, next, getState }) => {
         const {
+            actions: { setWalletSettingsLocalCurrency },
```

Comment:
if `changeWalletSettingsNetworks` is not used anymore, no need to keep it in extraDependencies

Reply:
f0796072864f0ae73f74e1d235b43c4dc83a2dfb